web-marketplace/src/components/organisms/ProjectDashboardBanner/ProjectDashboardBanner.tsx

```diff
@@ -0,0 +1,198 @@
+import EyeIcon from 'web-components/src/components/icons/EyeIcon';
+import { HorizontalDotsIcon } from 'web-components/src/components/icons/HorizontalDotsIcon';
+import TrashIcon from 'web-components/src/components/icons/TrashIcon';
+import ProjectPlaceInfo from 'web-components/src/components/place/ProjectPlaceInfo';
+import { Title } from 'web-components/src/components/typography';
+
+import {
+  CONVERT_TO_DRAFT,
+  DELETE_PROJECT,
+  EDIT,
+  MAX_ADDRESS_LENGTH_DESKTOP,
+  MAX_ADDRESS_LENGTH_MOBILE,
+  MIGRATE_PROJECT,
+  REGISTER_WITH_PROTOCOL,
+  UNTITLED_PROJECT,
+  VIEW,
+} from './ProjectDashboardBanner.constants';
+import { ProjectBannerProps } from './ProjectDashboardBanner.types';
+import { truncateEnd } from './ProjectDashboardBanner.utils';
+
+const ProjectDashboardBanner: React.FC<ProjectBannerProps> = ({
+  project,
+  canEdit,
+}) => {
+  const { _ } = useLingui();
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+  const navigate = useNavigate();
+
+  const truncatedPlace = truncateEnd(
+    project.place ?? '',
+    isMobile ? MAX_ADDRESS_LENGTH_MOBILE : MAX_ADDRESS_LENGTH_DESKTOP,
+  );
+
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const menuRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (!isMenuOpen) return;
+    const handleClickOutside = (event: MouseEvent) => {
+      if (menuRef.current && !menuRef.current.contains(event.target as Node)) {
+        setIsMenuOpen(false);
+      }
+    };
+    document.addEventListener('mousedown', handleClickOutside);
+    return () => {
+      document.removeEventListener('mousedown', handleClickOutside);
+    };
+  }, [isMenuOpen]);
+
+  const menuItems = [
+    {
+      label: _(REGISTER_WITH_PROTOCOL),
+      color: 'text-bc-neutral-700',
+      icon: <ClipboardIcon />,
+      action: () => setIsMenuOpen(false),
+    },
+    {
+      label: _(CONVERT_TO_DRAFT),
+      color: 'text-bc-neutral-700',
+      icon: <DraftDocumentIcon className="w-6 h-6" hasGradient />,
+      action: () => setIsMenuOpen(false),
+    },
+    {
+      label: _(MIGRATE_PROJECT),
+      color: 'text-bc-neutral-700',
+      icon: <ArrowDownIcon direction="next" fontSize="medium" hasGradient />,
+      action: () => setIsMenuOpen(false),
+    },
+    {
+      label: _(DELETE_PROJECT),
+      color: 'text-bc-neutral-700',
+      icon: <TrashIcon className="w-6 h-6 text-bc-red-400" />,
+      action: () => setIsMenuOpen(false),
+    },
+  ];
+
+  return (
+    <div className="relative w-full mt-20 ">
+      {/* Background Image Container */}
+      <div className="relative overflow-hidden border-solid border-[1px] border-bc-neutral-300 rounded-lg ">
+        {/* Original sharp background */}
+        <div
+          className="absolute inset-0 bg-cover bg-center"
+          style={{
+            backgroundImage: `url(${
+              project.imgSrc || '/default-project-image.jpg'
+            })`,
```

Comment:
ideally we would use our OptimizedImage component since images should come from s3 and can be optimized to serve the appropriate size instead of the full one